### DISCUSSION TITLE
Build not work

### DISCUSSION THREAD

**Discussion by ericoud** [rzeldent/esp32-smartdisplay#63] [Q&A]
hello, I am a beignet using platformIO...
I try to use the esp32-smartdisplay-demo. I clone it recursively as indicated. the library was installed (LVGL, Smartdisplay and audioI2s). When I compile the code I always have errors about smart display libraries not found:
...
and at the end:

.pio/libdeps/esp32-2432S028R/esp32_smartdisplay/src/esp_lcd_ili9341.c:11:37: fatal error: esp_lcd_panel_interface.h: No such file or directory

*********************************************************************************
* Looking for esp_lcd_panel_interface.h dependency? Check our library registry!
*
* CLI  > platformio lib search "header:esp_lcd_panel_interface.h"
* Web  > https://registry.platformio.org/search?q=header:esp_lcd_panel_interface.h
*
*********************************************************************************

and same error with  esp_arduino_version.h

I work on a MAC, perhaps it's the issue...

Sorry if my question seems to be irrelevant, in that case, just point me in the good direction.

Best regards

**Comment by rzeldent** [comment]
Hi eric,

Try to update the platform.

**Reply by ericoud** [reply]
I didn't fix the issue on my Mac, but using a windows PC it works like a charm.
Thank you for this great library.

**Comment by ericoud** [comment]
I use the last platformIO version v3.3.2.
I will do a test tomorrow with a windows PC...

**Comment by rzeldent** [comment]
Hi Eric,

Yes, I noticed I was also unable to build it on my mac. Very annoying that there are differences between pio -ci, platformIO regular builds but also differcences between the Mac and PC.

Now in the progress of (hopefully) aligning this a bit...

**Comment by rzeldent** [comment]
Works on mac with latest refactoring!
